docs/observability/telemetry-collection.md

```diff
@@ -0,0 +1,282 @@
+# Telemetry Collection
+
+Firebase Genkit Monitoring leverages Google Cloud metrics, traces, and logs to capture telemetry. This 
+document details which metrics, trace attributes, and logs will be collected and what you can expect for 
+latency, quotas, and cost.
+
+## Telemetry delay
+
+There may be a slight delay before telemetry for a particular execution of a
+flow is available in Firebase or Google Cloud. This is dependent on your export
+interval and by default within 5 to 10 minutes.
```

Comment:
```suggestion
There may be a slight delay before telemetry from a given invocation is available in Firebase. This is dependent on your export
interval (5 minutes by default).
```

Get rid of flow, simplify to Firebase?

Reply:
There may be processing delay on top of export delay, and I wasn't sure how to quantify or refer to that here.

Reply:
I don't think this is important. I accepted your edit.